.circleci/config/base.yml

```diff
@@ -1908,7 +1861,7 @@ jobs:
   osx-testing-x64:
     executor:
       name: macos
-      xcode: "13.3.0"
```

Comment:
Removing this will break build tools as it looks for xcode: ... to determine which version of xcode to use